### PR TITLE
Redcannary test

### PR DESCRIPTION
**Pull request by frack113**
Long time I did not run aurora vs redcannary atomic-red-team

### CODE REVIEW

**Comment by nasbench** [comment]
Could you please use permalinks instead of the master branch links :D

File: rules/windows/process_creation/proc_creation_win_susp_reg_add.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+title: Reg Add Suspicious Path 
+id: b7e2a8d4-74bb-4b78-adc9-3f92af2d4829
+status: experimental
+description: An adversary may use many registry path with reg.exe
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_susp_reg_add.yml:4]: Detects when an adversary uses the 'AppDataLow' subkeys as a place to store data as seen in the URSNIF phishing campaign

File: rules/windows/process_creation/proc_creation_win_susp_reg_add.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+title: Reg Add Suspicious Path
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_susp_reg_add.yml:1]: Reg Add Suspicious Path To AppDataLow

File: rules/windows/process_creation/proc_creation_win_susp_reg_add.yml (hunk 1 above)
Comment (frack113) [reply rules/windows/process_creation/proc_creation_win_susp_reg_add.yml]: The idea was to add more generic suspicious path. 
Will change If there are more added.

Comment (nasbench) [reply rules/windows/process_creation/proc_creation_win_susp_reg_add.yml]: Ah, sorry then didn't understand that. Good idea 👍

Comment (Neo23x0) [reply rules/windows/process_creation/proc_creation_win_susp_reg_add.yml]: Let's add suspicious folders right waway

Comment (Neo23x0) [reply rules/windows/process_creation/proc_creation_win_susp_reg_add.yml]: I'll try to find more with this query : https://www.google.de/search?q=inurl%3Aany.run+%22reg+add%22+%5CSoftware%5CAppDataLow%5C

Comment (Neo23x0) [reply rules/windows/process_creation/proc_creation_win_susp_reg_add.yml]: Okay, it's not that easy. We'll leave it as it is for now.

**Pull request merged**